clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -1024,7 +1026,15 @@ inline sycl::event matmul(descriptor_ptr handle, matmul_desc_ptr compute_desc,
 
   if (compute_desc->_epilogue != epilogue_t::nop) {
     ::dnnl::post_ops matmul_ops;
-    matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_relu, 0.f, 0.f);
+    if (compute_desc->_epilogue == epilogue_t::relu) {
+      matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_relu, 0.f, 0.f);
+    } else if (compute_desc->_epilogue == epilogue_t::bias) {
+      matmul_ops.append_binary(::dnnl::algorithm::binary_add, bias_md);
+    } else if (compute_desc-> epilogue == epilogue::gelu_aux_bias) {
+      matmul_ops.append_eltwise(::dnnl::algorithm::eltwise_gelu_erf, 0.f, 0.f);
+      matmul_ops.append_binary(::dnnl::algorithm::binary_add, bias_md);
+      compute_desc->set_attribute(matmul_desc_t::attribute::epilogue_aux_pointer, bias_mem);
```

Comment:
We need to copy the data in `bias_mem` to `_epilogue_aux_pointer`. Meanwhile, please also consider the data layout (row_major/col_major) and the leading dimension (CUBLASLT_MATMUL_DESC_EPILOGUE_AUX_LD).

And according to the cublaslt doc, `CUBLASLT_MATMUL_DESC_BIAS_POINTER` is also related to this epilogue. Please check the usage.

Reply:
I directly added matrix memcpy  to copy epilogue_aux_pointer contents from bias_mem pointer, using the epilogue_aux_ld is and new_ldc (bias ld). This usage is for matrix memcpy, and will try to see if tests are ok .
Please suggest in meantime.